cli/flox/src/commands/mod.rs

```diff
@@ -341,7 +333,7 @@ impl FloxArgs {
             floxhub_token,
             floxhub,
             catalog_client,
-            features,
+            features: config.features.clone().unwrap_or_default(),
```

Comment:
syle(nb): I think i prefer defining names and then just passing them onto the struct like before, but granted this whole block is rather inconsistent in that regard

Reply:
I'm inclined to leave it as-is because it's consistent with the other values that have fairly simple construction.